src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/commands/rc_command_request.h

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
the file path is  ...ion_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/commands/rc_command_request.h
why name rc_rpc_plugin under rpc_plugins?
is there any other plugins besides rpc_plugins under application_manager?
shall the rc_rpc_plugins be under application_manager directly? otherwise rc_plugin/sdl_plugin shall be enough to mean rc_rpc_plugin/sdl_rpc_plugin since it is already under rpc_plugins?